docs/user/pings/baseline.md

```diff
@@ -1,25 +1,63 @@
 # The `baseline` ping
 
 ## Description
-
 This ping is intended to provide metrics that are managed by the library itself, and not explicitly
 set by the application or included in the application's `metrics.yaml` file.
 
 ## Scheduling
-
-The `baseline` ping is automatically sent when the application is moved to the [background](index.md#defining-background-state).
+The `baseline` ping is automatically sent when the application is moved to the [background](index.md#defining-background-state) and it includes
+the following fields:
 
 ## Contents
-
----
-
-*Not yet implemented*
-
----
-
 | Field name | Type | Description |
 |---|---|---|
 | `duration` | Timespan | The duration, in seconds, of the last foreground session |
 | `locale` | String | The locale of the application |
 
-The `baseline` ping also includes the common [ping sections](pings.md) found in all pings.
+The `baseline` ping shall also include the common [ping sections](index.md#ping-sections) found in all pings.
+
+### Querying ping contents
+A quick note about querying ping contents (i.e. for https://sql.telemetry.mozilla.org):  Each metric
```

Comment:
Do we have autolink or should the url be wrapped in `<...>` to turn it into a link?

Reply:
Good point.  There's no auto-linking.  Making this into a proper link.